public/app/percona/dbaas/components/DBCluster/DBCluster.types.ts

```diff
@@ -36,6 +36,21 @@ export interface DBCluster {
   internetFacing?: boolean;
   sourceRanges?: string[];
   storageClass?: string;
+  backup?: DBaaSBackup;
+  restore?: DBaaSRestore;
+}
+
+interface DBaaSBackup {
```

Comment:
I think it makes sense to have two different types for the BE and FE and convert between them, this way we could keep consistent naming (camelCase) in the FE code as much as possible.

Reply:
thanks! fixed